docs/user-guide/01-blueprint-reference.md

```diff
@@ -436,9 +436,9 @@ An *optional* list of objects containing:
 - The `user` attribute is a **required** string and must match the name of a user in the image exactly.
 - The `key` attribute is a **required** string that contains the public key to be set for that user.
 
-*Warning: `key` expects the entire content of the public key file, traditionally `~/.ssh/id_rsa.pub` but any algorithm supported by the operating system in the image is valid*
+*Deprecated: you should prefer the [users](#additional-users) customization which allows you to add SSH keys to users. Not all frontends support this customization.*
```

Comment:
The anchor `#additional-users` needs to change.

Reply:
Why? The anchor is explicitly set to `#additional-users` right: https://github.com/osbuild/osbuild.github.io/blob/a2b421ec335f47a722955bd7f7f3b1716e6c1a08/docs/user-guide/01-blueprint-reference.md?plain=1#L475

Reply:
Right, my bad.  I thought the anchor would change with the heading.